src/main/resources/META-INF/services/org.apache.pirk.responder.wideskies.spi.ResponderPlugin

```diff
@@ -0,0 +1,5 @@
+org.apache.pirk.responder.wideskies.mapreduce.MapReduceResponder
```

Comment:
@ellisonanne the service-plugin model allows better extensibility than pure reflection and is a well known pattern so should be easier for others to develop new responders. 

This means that we keep platform designators and a developer for a new responder adds his/her own designator via `getPlatformName` which will be available when the jar is on the class path.  (I could actually put those into the command line but I won't as the config is in flux - todo I guess). 

It also allows a shorter command line 
```
-p spark
```
vs 
```
-l org.apache.pirk.responder.wideskies.spark.SparkResponderLauncher
```
with the same benefits.